requirements.txt

```diff
@@ -0,0 +1,8 @@
+beautifulsoup4==4.9.0
+bs4==0.0.1
+certifi==2020.4.5.1
+chardet==3.0.4
+idna==2.9
+requests==2.23.0
+soupsieve==2.0.1
+urllib3==1.25.9
```

Comment:
In your projects you should use `requirements.txt` file. It contains a list of all libraries (with their specific versions), required to run your project.

To make such file, you run:
```pip freeze > requirements.txt```

Then, when someone else will pull your project, that person can run:
``` pip install -r requirements.txt```
... and all dependencies will be installed.